src/App.tsx

```diff
@@ -38,79 +37,56 @@ export class App extends React.Component {
 
     return (
       <main className="section container">
-        {selectedGood === ''
-          ? (
-            <h1 className="title">No goods selected</h1>
-          ) : (
-            <h1 className="title is-flex is-align-items-center">
-              {`${this.state.selectedGood} is selected`}
-              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-              <button
-                onClick={this.handlerClear}
-                data-cy="ClearButton"
-                type="button"
-                className="delete ml-3"
-              />
-            </h1>
-          )}
+        {selectedGood === '' ? (
```

Comment:
We can simplify this condition by checking `!!selectedGood`. It will just look cleaner.

Reply:
same here